### DISCUSSION TITLE
v 0.0.63 install on CCU3

### DISCUSSION THREAD

**Discussion by WeisserWolf611** [thkl/hap-homematic#599] [Q&A] [answered]
Hello,

on my CCU3 there is v0.0.14 installed and I am not able to update to 0.0.63.
In readme I didn't found anything how I get an Update over npm directly on CCU3.
Is anybody here who could help?

**Comment by gerd2005** [comment, marked as answer]
Thats ok, you use the newest Releases. 0.14 is the installer. 0.63 is the running Software. (Or something like that)
